dss/operations/util.py

```diff
@@ -1,11 +1,16 @@
 import os
+import json
 import traceback
 import logging
 import typing
 
 from cloud_blobstore import BlobStore
 
 from dss.util.aws.clients import sts  # type: ignore
+from dss.util.aws.clients import es as es_client  # type: ignore
+from dss.util.aws.clients import secretsmanager as sm_client  # type: ignore
```

Comment:
Please avoid instantiating boto3 clients from this file. `dss/operations/util.py` will be pulled into many unrelated operational scripts.

Reply:
Thanks for this comment, it was helpful for clearing up how to decide what functionality to put where.